src/Command/CommandBase.php

```diff
@@ -668,6 +668,8 @@ protected function determineEnvironment(InputInterface $input, OutputInterface $
         } elseif ($input->getArgument('environmentId')) {
             $environmentId = $input->getArgument('environmentId');
             $chosenEnvironment = $this->getCloudEnvironment($environmentId);
+        } elseif ($ideEnvironment = $this->determineEnvironmentFromIdeContext($input, $output)) {
```

Comment:
Please restructure this to match the pattern for applications in the `else` case. In other words, move some of the logic up and out of the `determineEnvironmentFromIdeContext`.

Also, there's no reason to reference `Ide` in the name of this or any other method; a user should be able to set AH_CODEBASE_UUID in any environment and have this work. While you're at it, please rename this existing method to remove `Ide` as well: `getThisCloudIdeCloudAppUuid()`